assets/translations/main.json

```diff
@@ -310,6 +310,7 @@
 	"status.offline": "Offline",
 	"people-menu.title": "People",
 	"people-menu.change-name": "Change name",
+	"people-menu.avatar-color": "Avatar color",
```

Comment:
This be "Color", I think, given that the change message is "change-color". We don't use the word "Avatar" anywhere else.